build/webpack.dev.js

```diff
@@ -0,0 +1,67 @@
+const path = require('path');
```

Comment:
How does webpack get triggered?

Reply:
This is happening with `npm run dev`. Which will both start and http server and watch changes to recompile if the service worker file changes (maybe a but overkill but was very nice for development).